Apple M1 – Issue w/ @babel/runtime/helpers/interopRequireDefault.js error w/ NextJS, unable to use component

### Check that this is really a bug

- [X] I confirm

### Reproduction link

https://github.com/gavgrego/dead-cal

### Bug description

When trying to use component on a page with NextJS get an error related to `/home/runner/work/react-big-calendar/react-big-calendar/node_modules/@babel/runtime/helpers/interopRequireDefault.js`

`error - Error: Cannot find module '/home/runner/work/react-big-calendar/react-big-calendar/node_modules/@babel/runtime/helpers/interopRequireDefault.js'
Require stack:
- /Users/gavin/Sites/cal-dead/cal-dead/node_modules/react-big-calendar/lib/index.js
- /Users/gavin/Sites/cal-dead/cal-dead/.next/server/pages/index.js
- /Users/gavin/Sites/cal-dead/cal-dead/node_modules/next/dist/server/require.js
- /Users/gavin/Sites/cal-dead/cal-dead/node_modules/next/dist/server/load-components.js
- /Users/gavin/Sites/cal-dead/cal-dead/node_modules/next/dist/build/utils.js
- /Users/gavin/Sites/cal-dead/cal-dead/node_modules/next/dist/build/output/store.js
- /Users/gavin/Sites/cal-dead/cal-dead/node_modules/next/dist/build/output/index.js
- /Users/gavin/Sites/cal-dead/cal-dead/node_modules/next/dist/cli/next-dev.js
- /Users/gavin/Sites/cal-dead/cal-dead/node_modules/next/dist/bin/next`

### Expected Behavior

_No response_

### Actual Behavior

_No response_

### react-big-calendar version

0.36.4

### React version

0.36.4

### Platform/Target and Browser Versions

macOS M1

### Validations

- [X] Read the [docs](http://jquense.github.io/react-big-calendar/examples).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.
- [X] Make sure this is a react-big-calendar issue and not an implementation issue

### Would you like to open a PR for this bug?

- [ ] I'm willing to open a PR

--- a/.size-snapshot.json
+++ b/.size-snapshot.json
@@ -24,26 +24,26 @@
     }
   },
   "react-big-calendar.js": {
-    "bundled": 1583203,
-    "minified": 445033,
-    "gzipped": 137433
+    "bundled": 1583365,
+    "minified": 445101,
+    "gzipped": 137464
   },
   "react-big-calendar.min.js": {
-    "bundled": 283207,
-    "minified": 281758,
-    "gzipped": 88844
+    "bundled": 283536,
+    "minified": 281826,
+    "gzipped": 88860
   },
   "react-big-calendar.esm.js": {
-    "bundled": 206455,
-    "minified": 97326,
-    "gzipped": 25439,
+    "bundled": 200119,
+    "minified": 93592,
+    "gzipped": 24251,
     "treeshaked": {
       "rollup": {
-        "code": 69823,
-        "import_statements": 1103
+        "code": 66279,
+        "import_statements": 1777
       },
       "webpack": {
-        "code": 72947
+        "code": 70059
       }
     }
   }

--- a/babel.config.js
+++ b/babel.config.js
@@ -13,7 +13,7 @@ module.exports = function (api) {
           }),
         },
       ],
-      'react-app',
+      ['react-app', { absoluteRuntime: false }],
     ],
     plugins: [
       ['@babel/plugin-transform-runtime'],

